lib/standards/aria-roles.js

```diff
@@ -479,6 +479,7 @@ const ariaRoles = {
       'aria-expanded',
       'aria-orientation'
     ],
+    unsupportedAttrs: ['aria-required'],
```

Comment:
Are you sure this doesn't work on any radiogroup? I only tested radiogroup on a fieldset with VO. I'd like to have more test data before we make this change.